src/MessageTrait.php

```diff
@@ -138,20 +144,20 @@ public function hasHeader($header) : bool
      * If the header does not appear in the message, this method MUST return an
      * empty array.
      *
-     * @param string $header Case-insensitive header field name.
+     * @param string $name Case-insensitive header field name.
```

Comment:
Is there a specific reason why you would rename this? Just curious tho.
I am pretty sure that this might be a BC break aswell, as we already have PHP 8 support which added method calls with named arguments which will break if you change this.

But as we are implementing psalm, I'd prefer having the `@no-named-arguments` in here. Maybe we move this change to the next major even tho I don't understand why this change is here in the first place.